pkg/aspect/accessLogsManager.go

```diff
@@ -117,14 +118,29 @@ func (e *accessLogsWrapper) Close() error {
 }
 
 func (e *accessLogsWrapper) Execute(attrs attribute.Bag, mapper expr.Evaluator, ma APIMethodArgs) Output {
-	labels, err := evalAll(e.labels, attrs, mapper)
-	if err != nil {
-		return Output{Status: status.WithError(fmt.Errorf("failed to eval labels for log %s with err: %s", e.name, err))}
+
+	labels := make(map[string]interface{}, len(e.labels))
+	for name, exp := range e.labels {
+		v, err := mapper.Eval(exp, attrs)
+		if err == nil {
+			labels[name] = v
+			continue
+		}
+		if name == "timestamp" {
+			labels[name] = time.Now()
+		}
 	}
 
-	templateVals, err := evalAll(e.templateExprs, attrs, mapper)
-	if err != nil {
-		return Output{Status: status.WithError(fmt.Errorf("failed to eval template expressions for log %s with err: %s", e.name, err))}
+	templateVals := make(map[string]interface{}, len(e.templateExprs))
+	for n, expr := range e.templateExprs {
```

Comment:
Can we extract this to a local func so it can be shared between template and label expressions?